### ISSUE TITLE
ModuleNotFoundError: No module named 'requests'

### ISSUE THREAD

**Issue by heximing** [jjjchens235/bing-rewards#141]
It was working yesterday. Now it is not working.

I tried running requirements.txt again, and tried running setup.py again, entering email & password again from scratch, but still not working. 

C:\D\Python\WPy64-38100\python-3.8.10.amd64>python.exe C:\D\Python\bing-rewards-master\BingRewards\BingRewards.py
Traceback (most recent call last):
  File "C:\D\Python\bing-rewards-master\BingRewards\BingRewards.py", line 3, in <module>
    from src.rewards import Rewards
  File "C:\D\Python\bing-rewards-master\BingRewards\src\rewards.py", line 18, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'

**Comment by heximing** [comment]
By the way, I downloaded the newest codes, replaced the existing files, and ran setup.py again, but still not working.

**Comment by heximing** [comment]
I have been using this for months, no problem. But today it does not work.

**Comment by jjjchens235** [comment]
I have updated the requirements.txt file.
In your case, it might be easier just to run `python -m pip install requests` instead of doing all those steps again.

**Comment by briantho** [comment]
same error for me. I can't get past it. I installed requests via pip, and still am getting this error. here is the output from "pip install -r requirements.txt"

```

Requirement already satisfied: pip>=9.0.1 in c:\python310\lib\site-packages (from -r requirements.txt (line 1)) (21.3.1)
Requirement already satisfied: python-dateutil>=2.6.1 in c:\python310\lib\site-packages (from -r requirements.txt (line 2)) (2.8.2)
Requirement already satisfied: selenium>=3.8.0 in c:\python310\lib\site-packages (from -r requirements.txt (line 3)) (4.1.0)
Requirement already satisfied: setuptools>=28.8.0 in c:\python310\lib\site-packages (from -r requirements.txt (line 4)) (58.1.0)
Requirement already satisfied: six>=1.11.0 in c:\python310\lib\site-packages (from -r requirements.txt (line 5)) (1.16.0)
Requirement already satisfied: requests>=2.25.1 in c:\python310\lib\site-packages (from -r requirements.txt (line 6)) (2.27.1)
Requirement already satisfied: trio~=0.17 in c:\python310\lib\site-packages (from selenium>=3.8.0->-r requirements.txt (line 3)) (0.19.0)
Requirement already satisfied: urllib3[secure]~=1.26 in c:\python310\lib\site-packages (from selenium>=3.8.0->-r requirements.txt (line 3)) (1.26.8)
Requirement already satisfied: trio-websocket~=0.9 in c:\python310\lib\site-packages (from selenium>=3.8.0->-r requirements.txt (line 3)) (0.9.2)
Requirement already satisfied: idna<4,>=2.5 in c:\python310\lib\site-packages (from requests>=2.25.1->-r requirements.txt (line 6)) (3.3)
Requirement already satisfied: certifi>=2017.4.17 in c:\python310\lib\site-packages (from requests>=2.25.1->-r requirements.txt (line 6)) (2021.10.8)
Requirement already satisfied: charset-normalizer~=2.0.0 in c:\python310\lib\site-packages (from requests>=2.25.1->-r requirements.txt (line 6)) (2.0.10)
Requirement already satisfied: async-generator>=1.9 in c:\python310\lib\site-packages (from trio~=0.17->selenium>=3.8.0->-r requirements.txt (line 3)) (1.10)
Requirement already satisfied: outcome in c:\python310\lib\site-packages (from trio~=0.17->selenium>=3.8.0->-r requirements.txt (line 3)) (1.1.0)
Requirement already satisfied: sniffio in c:\python310\lib\site-packages (from trio~=0.17->selenium>=3.8.0->-r requirements.txt (line 3)) (1.2.0)
Requirement already satisfied: cffi>=1.14 in c:\python310\lib\site-packages (from trio~=0.17->selenium>=3.8.0->-r requirements.txt (line 3)) (1.15.0)
Requirement already satisfied: sortedcontainers in c:\python310\lib\site-packages (from trio~=0.17->selenium>=3.8.0->-r requirements.txt (line 3)) (2.4.0)
Requirement already satisfied: attrs>=19.2.0 in c:\python310\lib\site-packages (from trio~=0.17->selenium>=3.8.0->-r requirements.txt (line 3)) (21.4.0)
Requirement already satisfied: wsproto>=0.14 in c:\python310\lib\site-packages (from trio-websocket~=0.9->selenium>=3.8.0->-r requirements.txt (line 3)) (1.0.0)
Requirement already satisfied: cryptography>=1.3.4 in c:\python310\lib\site-packages (from urllib3[secure]~=1.26->selenium>=3.8.0->-r requirements.txt (line 3)) (36.0.1)
Requirement already satisfied: pyOpenSSL>=0.14 in c:\python310\lib\site-packages (from urllib3[secure]~=1.26->selenium>=3.8.0->-r requirements.txt (line 3)) (21.0.0)
Requirement already satisfied: pycparser in c:\python310\lib\site-packages (from cffi>=1.14->trio~=0.17->selenium>=3.8.0->-r requirements.txt (line 3)) (2.21)
Requirement already satisfied: h11<1,>=0.9.0 in c:\python310\lib\site-packages (from wsproto>=0.14->trio-websocket~=0.9->selenium>=3.8.0->-r requirements.txt (line 3)) (0.12.0)

```

this is what i see when trying to run bing-rewards.py

```
Traceback (most recent call last):
  File "C:\Users\mediacenter\Desktop\bing-rewards\bing-rewards\BingRewards\BingRewards.py", line 4, in <module>
    from src.rewards import Rewards
  File "C:\Users\mediacenter\Desktop\bing-rewards\bing-rewards\BingRewards\src\rewards.py", line 18, in <module>
    import requests
ModuleNotFoundError: No module named 'requests'
```

again this used to run fine some time ago. Now it's not working.

**Comment by jjjchens235** [comment]
An issue with your environment perhaps?
https://stackoverflow.com/questions/14295680/unable-to-import-a-module-that-is-definitely-installed

**Comment by briantho** [comment]
Hmm, interesting. I thought I left a response saying I solved it, but now I don't see that message anywhere... Wierd. 

Anyway, I was able to solve this issue. My issue turned out to be a leftover python install, that I wasn't aware of. Technically I had 2 python installs, one was under program files, and one was under c:/python310

I decided to remove both, and reinstalled python using the default path this time. After doing so, and redoing the pip requirements, it seems all is working as it should. Again, silly me, I had 2 versions of python in 2 different paths.
